operator/v1/types_console.go

```diff
@@ -143,8 +143,94 @@ type Capability struct {
 	Visibility CapabilityVisibility `json:"visibility"`
 }
 
+// ThemeType is the value of the CustomLogo Theme Type field that determines the theme mode in the console UI.
+type ThemeType string
+
+// ThemeType values
+const (
+	// ThemeTypeDark represents the dark theme for the console UI.
+	ThemeTypeDark ThemeType = "Dark"
+
+	// ThemeTypeLight represents the light theme for the console UI.
+	ThemeTypeLight ThemeType = "Light"
+
+	// ThemeTypeDefault represents the default theme for all unspecified themes.
+	ThemeTypeDefault ThemeType = "Default"
```

Comment:
So default means light and dark? But if I use default and dark, then default means light only?